### PR TITLE
DOC-2286: Removal and updates to various files.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2286

Changes:
* Updates to `7.0-release-notes.adoc` template outline.
* Updates to `release-notes.adoc-template` for future releases.
* Removal of `changelog.md` for tinymce-docs as agreed by @ShiridiGandham 
* Updates to `nav.adoc` file for sidebar.
* Updates to `pull_request_template.md` for tinymce-docs.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] `modules/ROOT/nav.adoc` has been updated (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by tiny-ben-tran — APPROVED** [review]

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -406,6 +406,21 @@
...
+**** xref:xref:7.0-release-notes.adoc#accompanying-premium-plugin-end-of-life-announcement[Accompanying Premium plugin end-of-life announcement]
+**** xref:xref:7.0-release-notes.adoc#accompanying-open-source-plugin-end-of-life-announcement[Accompanying open source plugin end-of-life-announcement]
+**** xref:xref:7.0-release-notes.adoc#accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
+**** xref:xref:7.0-release-notes.adoc#improvements[Improvements]
+**** xref:xref:7.0-release-notes.adoc#additions[Additions]
+**** xref:xref:7.0-release-notes.adoc#changes[Changes]
+**** xref:xref:7.0-release-notes.adoc#bug-fixes[Bug fixes]
+**** xref:xref:7.0-release-notes.adoc#security-fixes[Security fixes]
+**** xref:xref:7.0-release-notes.adoc#deprecated[Deprecated]
+**** xref:xref:7.0-release-notes.adoc#known-issues[Known issues]
```
Comment (MitchC1999) [inline modules/ROOT/nav.adoc:423]:
```suggestion
**** xref:7.0-release-notes.adoc#new-premium-plugin<s>[New Premium plugin<s>]
**** xref:7.0-release-notes.adoc#new-open-source-plugin<s>[New Open Source plugin<s>]
**** xref:7.0-release-notes.adoc#accompanying-premium-plugin-changes[Accompanying Premium plugin changes]
**** xref:7.0-release-notes.adoc#accompanying-premium-plugin-end-of-life-announcement[Accompanying Premium plugin end-of-life announcement]
**** xref:7.0-release-notes.adoc#accompanying-open-source-plugin-end-of-life-announcement[Accompanying open source plugin end-of-life-announcement]
**** xref:7.0-release-notes.adoc#accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
**** xref:7.0-release-notes.adoc#improvements[Improvements]
**** xref:7.0-release-notes.adoc#additions[Additions]
**** xref:7.0-release-notes.adoc#changes[Changes]
**** xref:7.0-release-notes.adoc#bug-fixes[Bug fixes]
**** xref:7.0-release-notes.adoc#security-fixes[Security fixes]
**** xref:7.0-release-notes.adoc#deprecated[Deprecated]
**** xref:7.0-release-notes.adoc#known-issues[Known issues]
```
Something you might've missed.

**Review by ltrouton — APPROVED** [review]

**Review by MitchC1999 — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,15 +1,14 @@
-Ticket: DOC-<num>, <ticket-title-here>.
+Ticket: DOC-<num>
```
Comment (MitchC1999) [inline .github/pull_request_template.md:1]:
```suggestion
Ticket: DOC-<num>

Site: http://docs-<hotfix/feature>-7-doc-<num>.staging.tiny.cloud/docs/tinymce/latest/
```
Might be cool to have a link to the dev site in the PR, while we're making changes.

Comment (kemister85) [reply .github/pull_request_template.md:1]: as in the staging branch?

**Review by FarzadHayat — APPROVED** [review]

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
+
+Site: http://docs-<hotfix/feature>-7-doc-<num>.staging.tiny.cloud/docs/tinymce/latest/
```
Comment (kemister85) [inline .github/pull_request_template.md:3]:
```suggestion
Site: http://docs-<hotfix/feature>-70-doc-<num>.staging.tiny.cloud/docs/tinymce/latest/
```

**Pull request merged**